lnbits/core/views/generic.py

```diff
@@ -414,29 +390,3 @@ async def hex_to_uuid4(hex_value: str):
         raise HTTPException(
             status_code=HTTPStatus.BAD_REQUEST, detail=str(exc)
         ) from exc
-
-
-async def toggle_extension(extension_to_enable, extension_to_disable, user_id):
```

Comment:
this logic is moved to: `/api/v1/extension/{ext_id}/enable` and `/api/v1/extension/{ext_id}/disable`